src/prod/override.yaml

```diff
@@ -65,7 +70,6 @@ harness:
       tolerations: []
 
   et:
-    # -- Enabled will deploy ET to your cluster
     enabled: false
```

Comment:
This `enabled` config can be removed I think

Reply:
good catch, fixed